submarine-workbench/submarine-server/src/main/java/org/apache/submarine/database/entity/TeamMember.java

```diff
@@ -45,4 +57,17 @@ public Integer getInviter() {
   public void setInviter(Integer inviter) {
     this.inviter = inviter;
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
```

Comment:
In the java guide [4.1.1](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used), this is not acceptable.